t/ui/14-dashboard-parents.t

```diff
@@ -138,7 +138,7 @@ subtest 'filtering subgroups' => sub {
     $ele->send_keys(Selenium::Remote::WDKeys->KEYS->{end}, '0');    # appended
     $driver->find_element('#filter-apply-button')->click();
     wait_for_ajax();
-    $url .= '?group=Test%20parent%20%2F%20.*%20test%24&default_expanded=1&limit_builds=30&time_limit_days=140';
+    $url .= '?group=Test%20parent%20%2F%20.*%20test%24&default_expanded=1&limit_builds=30&time_limit_days=140&interval=';
```

Comment:
this became too long now, see https://app.circleci.com/pipelines/github/os-autoinst/openQA/4186/workflows/b1c06fc4-f9b1-42df-bd94-267a323f48e7/jobs/39997

Better concatenate strings in individual statements, e.g.:

```
$url .= '?group=Test%20parent%20%2F%20.*%20test%24';
$url .= '&default_expanded=1&limit_builds=30&time_limit_days=140&interval=';
```